assets/blocks/lesson-actions/lesson-actions-block/lesson-actions-editor.scss

```diff
@@ -26,18 +23,15 @@
 }
 
 .wp-block[data-type="sensei-lms/lesson-actions"] {
-
-	&.is-selected .wp-block-sensei-lms-button-next-lesson__wrapper {
-		position: static;
+	&.is-selected .wp-block-sensei-lms-button-next-lesson__wrapper,
+	&.is-selected .wp-block-sensei-lms-button-complete-lesson__wrapper {
 		&:before {
 			position: absolute;
 			content: '';
-			left: 12px;
-			right: 12px;
-			border-top: var(--wp-admin-border-width-focus, 1.5px) solid var(--wp-admin-theme-color, rgba(66,88,99,.4));
-			margin-top: -9px;
+			top: 6px;
+			bottom: 6px;
+			margin-left: -9px;
 			pointer-events: none;
 		}
 	}
```

Comment:
This can be removed too if there is no selection separator for the groups.

Reply:
Woops! Didn't realise that this part was for the separator.